### ISSUE TITLE
Right click -> comment (feature)

### ISSUE THREAD

**Issue by gtu78aed87t** [files-community/Files#11594]
### What feature or improvement do you think would benefit Files?

I wish it was possible to right click and comment on files.

### Requirements

n/a

### Files Version

v2.4.40

### Windows Version

all

### Comments

n/a

**Comment by Josh65-2201** [comment]
You can access file comments by Right clicking > Properties > Details > Comment section.

Is your request to have there be a shortcut to that in the right click menu?

**Comment by yaira2** [comment]
@gtu78aed87t

**Comment by gtu78aed87t** [comment]
Yes, a shortcut to that in the right click menu is what I ment, but also a hotkey would be great too (perhaps it already exists, but missed me? That would be great!). 
Personally, I like to comment on files to remember different stuff, and would just like there to be less friction doing that.

**Comment by yaira2** [comment]
We don't have plans to add shortcut for the details page, but we are working on keyboard shortcuts for opening the properties window.
